packages/internals/funjucks/src/filters/await.test.js

```diff
@@ -0,0 +1,45 @@
+const {expect} = require('../../../../../test/helpers');
+const factory = require('./await');
+
+describe('await', function () {
+  it('exports a function', function () {
+    expect(factory).to.be.a('function');
+  });
+
+  it('returns an object with the expected properties', function () {
+    const filter = factory();
+    expect(filter.name).to.equal('await');
+    expect(filter.async).to.equal(true);
+    expect(typeof filter.filter).to.equal('function');
+  });
```

Comment:
(Lines 5-14)
These look like they're pretty consistent across the filters, could be extracted to common filter tests?